repondeur/tests/fetch/test_liasse_xml.py

```diff
@@ -0,0 +1,128 @@
+from datetime import date
+from pathlib import Path
+
+
+SAMPLE_LIASSE = Path(__file__).parent.parent / "sample_data" / "liasse.xml"
+
+
+def test_import_liasse_xml():
+    from zam_repondeur.fetch.an.liasse_xml import import_liasse_xml
+    from zam_repondeur.fetch.models import Amendement
+
+    with SAMPLE_LIASSE.open(mode="rb") as sample_file:
+        amendements = import_liasse_xml(sample_file)
```

Comment:
Proposition: `amendements = import_liasse_xml(SAMPLE_LIASSE.read_bytes())`

Reply:
En fait il faut : `amendements = import_liasse_xml(SAMPLE_LIASSE.open(mode="rb"))`

Reply:
Are you sure you cannot do that: https://github.com/betagouv/zam/pull/90/files#diff-1e0d3a29fb9866f9d91e804a2cce1c8fR12 ?

Reply:
That's because `import_liasse_xml()` wants a file-like object, not the file's contents (otherwise that would be a good way to do it indeed).

Reply:
Ooooh OK, got it!